src/assets/scripts/resources-utils.js

```diff
@@ -0,0 +1,326 @@
+			<div class='card-tags'>
+		`;
+		if (resource.topics) {
+			resource.topics.forEach(topicValue => {
+				let found = resourceTopics.find(topicObj => topicObj.value === topicValue);
+				resourcesHtml += `    <div class='card-tag'>
+					<svg role="presentation"><use xlink:href="#topic" /></svg>
+					<p>${found.label}</p>
+				</div>`;
+			});
+		}
+		if (resource.types) {
+			resource.types.forEach(typeValue => {
+				let found = resourceTypes.find(typeObj => typeObj.value === typeValue);
+				resourcesHtml += `    <div class='card-tag'>
+					<svg role="presentation"><use xlink:href="#type" /></svg>	
+					<p>${found.label}</p>
+				</div>`;
+			});
+		}
+		resourcesHtml += `
+				</div>
+				<div class='card-description'>
+					<p>${resource.description}</p>
+				</div>
+				${resource.publishedYear ? `<div class='card-publishedYear'><p>Published in ${resource.publishedYear}</p></div>` : ''}
+				<div class='card-link'><a rel='external' href='${resource.link}'>Visit ${resource.title}${resourceLink.host === hostURL ? '' : '<svg role="presentation"><use xlink:href="#external" /></svg>'}</a></div>
+				</div>
+				${resource.thumbnailImage ? `<div class='card-image'><img src=${resource.thumbnailImage} alt=${resource.thumbnailAltText ? resource.thumbnailAltText : `Thumbnail image for ${resource.title}`}></div>` : ''}
+			</div>
+		</div>`;
+	});
+
+	document.querySelector('.content').innerHTML += resourcesHtml;
+}
+
+/*
+ * Render the pagination.
+ * @param {Object} pagination - A pagination object to be rendered.
+* @return directly add the pagiination html to the pagination selector.
+  */
+function renderPagination(pagination) { // eslint-disable-line no-unused-vars
+	let paginationHtml = `
+	<h2 id='pagination' class='visually-hidden'>resources page navigation</h2>
+		<ul class='container'>`;
+
+	if (pagination.href.previous) {
+		paginationHtml += `
+			<li><a href='${ pagination.href.previous }'><svg><use xlink:href='#previous' /></svg><span class='visually-hidden'>previous</span></a></li>`;
+	}
+	pagination.pages.forEach((page, pageNumber) => {
+		paginationHtml += `
+			<li><a href='${ pagination.hrefs[pageNumber] }'`;
+		if (pagination.pageNumber === pageNumber) {
+			paginationHtml += ' aria-current="page"';
+		}
+		paginationHtml += `>${ pageNumber + 1 }</a></li>`;
+	});
+	if (pagination.href.next) {
+		paginationHtml += `
+			<li><a href='${ pagination.href.next }'><svg><use xlink:href='#next' /></svg><span class='visually-hidden'>next</span></a></li>`;
+	}
+	paginationHtml += `
+		</ul>`;
+	document.querySelector('.pagination').innerHTML = paginationHtml;
+}
+
+/*
+ * Set focus back on items before the refresh by filter changes
+*/
+function restoreFocus() {
+	if (localStorage.getItem('setFocusOn')) {
+		const focusElement = document.querySelector(localStorage.getItem('setFocusOn'));
+		if (focusElement && focusElement.focus) {
+			focusElement.focus();
+		}
+		localStorage.removeItem('setFocusOn');
+	}
+}
```

Comment:
Add an EOF at the end of `resources-utils.js` so github interface doesn't report this warning.